lib/containers/container_images.pm

```diff
@@ -105,7 +105,9 @@ sub test_opensuse_based_image {
     # It is the right version
     if ($distri eq 'sle') {
         my $pretty_version = $version =~ s/-SP/ SP/r;
-        validate_script_output("$runtime container run --entrypoint '/bin/bash' --rm $image -c 'cat /etc/os-release'", sub { /PRETTY_NAME="SUSE Linux Enterprise Server $pretty_version"/ });
+        record_info "Validating That $image has $pretty_version on /etc/os-release";
```

Comment:
http://kepler.suse.cz/tests/2575#step/podman_image/54

Maybe we can move the long message to the second argument of `record_info`. WDYT ?